src/cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
 /* eslint-disable id-length, no-process-exit */
+require("regenerator-runtime/runtime")
 const meow = require('meow')
-const readFileStream = require('./readFileStream')
 const { parse, evaluate } = require('groq-js')
 const getStdin = require('get-stdin')
 const chalk = require('chalk')
+const ndjson = require('ndjson')
 require('./gracefulQuit')
 const colorizeJson = require('./colorizeJson')
-const fromUrl = require('./fromUrl')
+const S2A = require('stream-to-async-iterator').default
 const cli = meow(
   `
 Usage
   ${chalk.green(`$ groq '*[<filter>]{<projection>}'`)}
   ${chalk.grey(`# Remember to alternate quotation marks inside of the query`)}
 
 Options
-  ${chalk.green(`--file  ./path/to/file`)}
-  ${chalk.green(`--url https://aniftyapi.dev/endpoint`)}
-  ${chalk.green(
-    `--primary results | The primary element with the array you want to query`
-  )}
-  ${chalk.green(`--pretty colorized JSON output [Default: false]`)}
+  ${chalk.green(`-i, --input   One of: ndjson, json, null`)}
+  ${chalk.green(`-o, --output  One of: ndjson, json, pretty`)}
+  ${chalk.green(`-p, --pretty  Shortcut for --output=pretty`)}
+
+Input formats
+  ${chalk.green(`json`)}      Reads a JSON object from stdin. Available as @ in query.
+  ${chalk.green(`ndjson`)}    Reads a JSON stream from stdin. Available as * in query.
+  ${chalk.green(`null`)}      Reads nothing.
+
+Output formats
+  ${chalk.green(`json`)}      Formats the output as JSON.
+  ${chalk.green(`pretty`)}    Pretty prints the output.
+  ${chalk.green(`ndjson`)}    Streams the result as NDJSON.
 
 Examples
-  ${chalk.grey(`# Query data in a ndjson-file`)}
-  ${chalk.green(`$ groq '*[_type == "post"]{title}' --file ./blog.ndjson`)}
+  ${chalk.grey(`# Query data in a file`)}
+  ${chalk.green(`$ cat blog.json | groq 'count(posts.length)' `)}
```

Comment:
Is it `posts.length` and not just `posts`?

Reply:
Yes. Fixed now.